Bottom: 606449b4b2d3572f12aefed022fabdb8bc595554
Top:    765c0028f2a0bb6764bcdcb05c3a4a0e39176aea
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-05 16:02:02 +0000

Refresh of disable-debug

---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 364d875..ba5c857 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -1504,7 +1504,7 @@ object3d_from_copper_layers_within_area (POLYAREA *area)
                               copper_appearance,
                               NULL));
 
-    break;
+    //break;
   }
 
 
@@ -1516,10 +1516,10 @@ object3d_from_copper_layers_within_area (POLYAREA *area)
 //  gui->invalidate_all ();
 
   /* ASSUME THERE IS A POLYGON WHERE WE KNOW WE PUT ONE... */
-  ((PolygonType *)PCB->Data->Layer[1].Polygon->data)->Clipped = info.poly;
-  gui->invalidate_all ();
+//  ((PolygonType *)PCB->Data->Layer[1].Polygon->data)->Clipped = info.poly;
+//  gui->invalidate_all ();
 
-//  poly_Free (&info.poly);
+  poly_Free (&info.poly);
 
   return objects;
 }
